test/app.test.js

```diff
@@ -15,6 +15,10 @@ var describe = require('./util/describe');
 var expect = require('chai').expect;
 var it = require('./util/it');
 
+beforeEach(function() {
+  this.app = app = loopback();
```

Comment:
Same comment as above, move this code to `describe` block and define `app` as a describe-local variable.